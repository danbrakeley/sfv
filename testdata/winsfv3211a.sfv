; Generated by WIN-SFV32 v1.1a on 2009-10-01 at 12:07.43
;
;      3457071  12:07.42 2009-10-01 01 Palm Trees.wav
;      6143256  12:07.42 2009-10-01 02 Loving Your Dog.wav
;      9586782  12:07.42 2009-10-01 03 Laundry Day.wav
;      5823465  12:07.42 2009-10-01 04 Goodbye.wav
;      6586736  12:07.42 2009-10-01 05 Palm Trees Tavern.wav
;      2438939  12:07.42 2009-10-01 06 Good Golly Grandma.wav
;      9084801  12:07.42 2009-10-01 07 Roku.wav
;      3204596  12:07.42 2009-10-01 08 Ninja Child.wav
;      3842908  12:07.42 2009-10-01 09 Alien Resident.wav
;      6047853  12:07.42 2009-10-01 10 Forgotten....wav
;      5408328  12:07.42 2009-10-01 11 Theme of ''GRAPE APE''.wav
;      2145672  12:07.42 2009-10-01 12 Theme (Piano Version).wav
01 Palm Trees.wav 29493C83
02 Loving Your Dog.wav A6A6736B
03 Laundry Day.wav 2B62EA05
04 Goodbye.wav F8CEFA93
05 Palm Trees Tavern.wav 5C2C7A5F
06 Good Golly Grandma.wav 7ABCC091
07 Roku.wav FB8C4CA6
08 Ninja Child.wav ED4EF8A1
09 Alien Resident.wav 9437529E
10 Forgotten....wav AB228ED3
11 Theme of ''GRAPE APE''.wav A2402EB3
12 Theme (Piano Version).wav 986DFACA
