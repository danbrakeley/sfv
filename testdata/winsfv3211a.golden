01 Palm Trees.wav
29493c83
02 Loving Your Dog.wav
a6a6736b
03 Laundry Day.wav
2b62ea05
04 Goodbye.wav
f8cefa93
05 Palm Trees Tavern.wav
5c2c7a5f
06 Good Golly Grandma.wav
7abcc091
07 Roku.wav
fb8c4ca6
08 Ninja Child.wav
ed4ef8a1
09 Alien Resident.wav
9437529e
10 Forgotten....wav
ab228ed3
11 Theme of ''GRAPE APE''.wav
a2402eb3
12 Theme (Piano Version).wav
986dfaca
